docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -41,6 +41,9 @@ join the cluster. Generate a short-lived join token and save it for example
 in `/tmp/token`:
 
 ```code
+# Log in to your cluster with tsh so you can use tctl.
+# You can also run tctl on your Auth Service host.
```

Comment:
It's not totally clear that when running on the auth service host the `tsh login` step is not necessary.